src/components/EventCard/EventCard.js

```diff
@@ -114,23 +114,31 @@ function EventCard({
 	return (
 		<Card
 			raised
-			elevation={isHover ? 11 : 6}
+			// elevation={isHover ? 11 : 6}
```

Comment:
hmm let's try to use this `elevation` prop rather than the translating the card vertically because elevation also changes the drop shadow I think